packages/react-vapor/src/components/logoCard/LogoCard.tsx

```diff
@@ -63,6 +63,9 @@ export class LogoCard extends React.Component<ILogoCardProps & React.HTMLProps<H
         );
         const logoIconClassName = classNames(DEFAULT_LOGO_ICON_CLASSNAME, DEFAULT_LOGO_ICON_SIZE);
         const descriptionClassName = classNames(this.props.badges.length ? 'ml1' : '');
+        const logoCardContentClassName = classNames(
+            !this.props.badges.length && !this.props.description ? 'logo-card-content-center' : 'logo-card-content'

```

Comment:
Just curious, would it work if you simply evaluate if the badge prop is not undefined without checking the length? 

like so: `!this.props.badges && !this.props.description ? 'logo-card-content-center : 'logo-card-content'`

Reply:
I think I need to check the length because the static defaultProps returns `badges` as an empty array type. I tried your solution to be sure and it fails the UTs

Reply:
Oh, make sense!